bigtable-hbase-dataflow/pom.xml

```diff
@@ -138,6 +138,36 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+
```

Comment:
This plugin should be wrapped in the "release" profile rather than run for every build.  I'll do that post merge.